src/commands/monitors/run.rs

```diff
@@ -13,10 +12,10 @@ pub fn make_command(command: Command) -> Command {
     command
         .about("Wraps a command")
         .arg(
-            Arg::new("monitor")
-                .help("The monitor ID")
-                .required(true)
-                .value_parser(Uuid::parse_str),
+            Arg::new("monitor_slug")
+                .value_name("monitor-slug")
```

Comment:
This is a breaking change and we need to preserve a hidden alias that'd still support UUID.

Reply:
all slugs on the backend match existing UUIDs, it is fully backwards compatible.

Reply:
See https://github.com/getsentry/sentry/issues/45166

Reply:
Ah, perfect 😌